WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/Package Creation/ShippingLabelAddNewPackageViewModel.swift

```diff
@@ -5,6 +5,9 @@ import SwiftUI
 ///
 final class ShippingLabelAddNewPackageViewModel: ObservableObject {
     @Published var selectedIndex: Int
+    var selectedView: SelectedIndex {
+        SelectedIndex(rawValue: selectedIndex) ?? .customPackage
+    }
```

Comment:
The return type `SelectedIndex` here sounds a bit ambiguous to me, since the variable name is `selectedView` and I'm expecting a view instead. How about renaming the enum to something like `PackageViewType`?

Reply:
I agree. I tried renaming it before and just ended up confusing myself, but `PackageViewType` is clearer than what I had thought of. Thanks!